deploy/team-api-update.js

```diff
@@ -21,11 +34,11 @@ function spawn(action_description, path, args) {
   });
 }
 
-function bundleInstall() {
+TeamApiUpdater.prototype.bundleInstall = function bundleInstall() {
```

Comment:
The function definition in this case should be anonymous, no?

```TeamApiUpdater.prototype.bundleInstall = function () { ...```

(same thing down below on line 41)